app/web/src/components/Footer/Footer.tsx

```diff
@@ -0,0 +1,198 @@
+import { Link as MuiLink, Typography } from "@material-ui/core";
+import classNames from "classnames";
+import Button from "components/Button";
+import { COPYRIGHT } from "components/Footer/constants";
+import StyledLink from "components/StyledLink";
+import { TERMS_OF_SERVICE } from "features/auth/constants";
+import { SHOW_ALL_EVENTS } from "features/communities/constants";
+import { ReactNode } from "react";
+import { Link } from "react-router-dom";
+import {
+  blogURL,
+  contributeRoute,
+  donationsRoute,
+  eventsRoute,
+  faqURL,
+  forumURL,
+  foundationURL,
+  planURL,
+  teamURL,
+  tosRoute,
+  townHallURL,
+} from "routes";
+import makeStyles from "utils/makeStyles";
+
+import {
+  ABOUT,
+  BLOG,
+  COMMUNITY,
+  DONATE,
+  EVENTS,
+  FAQ,
+  FORUM,
+  FOUNDATION,
+  OUR_PLAN,
+  OUR_TEAM,
+  TOWN_HALL,
+  VOLUNTEER,
+} from "../../constants";
+
+interface FooterProps {
+  maxWidth: string | number;
+  paddingInline: string | number;
+}
+
+const useStyles = makeStyles((theme) => ({
+  root: {
+    display: "flex",
+    flexDirection: "column",
+    width: "100%",
+  },
+  containerPadding({ maxWidth, paddingInline }: FooterProps) {
+    return {
+      maxWidth,
+      paddingInlineStart: paddingInline,
+      paddingInlineEnd: paddingInline,
+    };
+  },
+  upperOuterContainer: {
+    display: "flex",
+    justifyContent: "center",
+    marginBlockStart: theme.spacing(2),
+    paddingBlockStart: theme.spacing(3),
+    paddingBlockEnd: theme.spacing(3),
+    borderTop: `solid 1px ${theme.palette.divider}`,
+    backgroundColor: theme.palette.background.paper,
+  },
+  upperContainer: {
+    width: "100%",
+    display: "grid",
+    rowGap: theme.spacing(1),
+    columnGap: theme.spacing(1),
+    gridTemplateColumns: "auto auto",
+    [theme.breakpoints.up("sm")]: {
+      gridTemplateColumns: "auto auto auto auto",
```

Comment:
```suggestion
      gridTemplateColumns: "repeat(4, auto)",
```
I think this would also work and slightly shorter to write